roles/build_openstack_packages/defaults/main.yml

```diff
@@ -70,6 +70,7 @@ cifmw_bop_change_list: []
 cifmw_bop_release_mapping:
   master: master
   antelope: unmaintained/2023.1
+  epoxy: stable/2025.1
```

Comment:
This change is unrelated to your intended modification.
Please remove it from this PR.

Reply:
Moved this to a seperate pr https://github.com/openstack-k8s-operators/ci-framework/pull/2950